src/CsvTextEditor/ViewModels/MainViewModel.cs

```diff
@@ -35,34 +39,23 @@ public MainViewModel(IProjectManager projectManager)
         public Project Project { get; set; }
 
         #region Methods
-        protected override Task InitializeAsync()
+        protected override async Task InitializeAsync()
         {
-            _projectManager.ProjectActivatedAsync += OnProjectActivated;
+            await base.InitializeAsync();
 
-            return base.InitializeAsync();
+            _projectManager.ProjectActivationAsync += OnProjectActivationAsync;
         }
```

Comment:
why not? I think we should initialize base class and then do child class stuff. hm... I look into Rantt you can find the same code in MainViewModel... May be I don't get it... what should i do if i want to initialize base class first in this situation?